server/utils/isHtmlRequest.spec.js

```diff
@@ -0,0 +1,34 @@
+const { expect } = require('chai')
+const isHtmlRequest = require('./isHtmlRequest')
+require('mocha')
+
+describe('isHtmlRequest', () => {
+  it('should return true when html is requested', () => {
+    [
+      'developers/index.html',
+      '/developers/index.html#fragment',
+      '/getting-started/index.html?foo=bar#fragment',
+      'https://algorithmia.com/developers/index.html',
+      'https://algorithmia.com/developers/index.html?foo=bar#2',
+      'https://algorithmia.com/developers.html?foo=bar#2',
+      'https://algorithmia.com/developers/getting-started/index.html?foo=bar#fragment',
+      'https://algorithmia.com/developers/getting-started.html?foo=bar#fragment',
+    ].forEach(path => {
+      expect(isHtmlRequest(path)).to.be.true
+    })
+  })
+
+  it('should return false for all other requests', () => {
+    [
+      '',
+      'developers/',
```

Comment:
Okay, so we still want the cache header set to a year on things like `/developers/`?

Reply:
Technically this won't happen, as a request to `/developers` will be updated to a request to `/developers/index.html` behind the scenes. This is more just for the sake of testing my regex.

Reply:
Ah, I guess we add the `.html` to these before we check for it in the server file.